cost/azure/idle_compute_instances/azure_idle_compute_instances.pt

```diff
@@ -98,6 +98,7 @@ datasource "ds_azure_virtualmachines" do
 	path join(["/subscriptions/", val(iter_item,"subscriptionId"), "/providers/Microsoft.Compute/virtualMachines"])
     query "api-version","2019-03-01"
     header "User-Agent", "RS Policies"
+    ignore_status [403,404]
```

Comment:
The other policies POL-311 and POL-310 didn't not include ignore_status 403.  Is 403 needed,?  If so, why wasn't it included in the other policies?

Reply:
I thought 403 may also occur since we are iterating subscriptions ID. I have removed 403 and committed the changes.